admiral/pkg/clusters/registry_test.go

```diff
@@ -62,7 +62,7 @@ func TestDeleteCacheController(t *testing.T) {
 
 	w := RemoteRegistry{
 		RemoteControllers: make(map[string]*RemoteController),
-		StartTime: time.Now(),
+		StartTime:         time.Now(),
```

Comment:
nitpick, formatting issues.

Reply:
Seems to be the right formatting now. Was messed up before.